firebase-crashlytics/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 # Unreleased
-
 * [changed] Include more details about app processes in reports.
 
+# 18.5.1
+* [fixed] Fixed Flutter and Unity on-demand fatal `setUserIdentifier` behaviour. Github
```

Comment:
This is completely different than the release notes. Did they get drastically changed in the CL?